clustering/tasks.py

```diff
@@ -0,0 +1,25 @@
+from django.core.cache import cache
+from celery.decorators import task
+
+from clustering_settings import CLUSTERING_CACHE_TIME, CLUSTERING_PENDING_CACHE_TIME
+from . import CLUSTERING_RESULT_STATUS_PENDING, CLUSTERING_RESULT_STATUS_FAILED
+
+
+@task(name="cluster_sounds")
+def cluster_sounds(cache_key_hashed, sound_ids, features):
+    # this import would be better outside the function, but did not work
```

Comment:
This import ensures that the engine is imported after it is re-assigned in __init__.py
There should be a better way to do it to avoid multiple imports that can decrease performance.

Simply importing outside of the scope of the function did not work and gives None value for engine.